srcs/vm/vm.h

```diff
@@ -38,6 +38,11 @@
 # define DEBUG_PRINT                1
 # define DEBUG_FILE                 "debug.log"
 
+#define ARG_TYPE_REG			1
+#define ARG_TYPE_DIR			2
+#define ARG_TYPE_IND			3
+#define ARG_TYPE_NONE			0
```

Comment:
In OP.H hebben we dit al staan:

![image](https://user-images.githubusercontent.com/591280/90023876-8746a780-dcb4-11ea-87fb-b2ed169b5d86.png)